components/x-topic-search/src/TopicSearch.jsx

```diff
@@ -43,8 +43,26 @@ const topicSearchActions = withActions(({ minSearchLength = 2, maxSuggestions =
 	}
 }));
 
-const TopicSearch = topicSearchActions(({ searchTerm, showResult, result, actions, isLoading, csrfToken }) => (
-	<div className={ classNames(styles['container']) }>
+const TopicSearch = topicSearchActions(({ searchTerm, showResult, result, actions, isLoading, csrfToken, followedTopics }) => {
+
+	document.body.addEventListener('x-follow-button', event => {
```

Comment:
I think the action function this lives in will get called multiple times, meaning multiple event listeners will be added.

I don't think there's a better place to put this, so I think the solution is to extract the event handler function, and then remove the listener before adding it. Then only one such listener will ever exist. Something like:

```javascript
const handleFollowEvent = event => {
   // unchanged...
};

document.body.removeEventListener('x-follow-button', handleFollowEvent);
document.body.addEventListener('x-follow-button', handleFollowEvent);
```